### ISSUE TITLE
Suggested changes to pandas lesson

### ISSUE THREAD

**Issue by deniederhut** [dlab-berkeley/introduction-to-pandas#5]
Possible things to consider for intermediate/advanced pandas class:
- [ ]  reading/writing to formats other than csv
- [ ]  reading data in by chunks
- [ ]  multi-indexing  and its methods
- [ ]  sparse matrices and their methods
- [ ]  ways to speed up mathy things (lots of %timeit)
- [ ]  parallelized methods for pandas

We can both also look up other pandas materials and think of other things we might like to add.

**Comment by juanshishido** [comment]
Thanks! This looks like a great start.

**Comment by juanshishido** [comment]
I'll start a concurrent list of other _potentially_ useful topics/methods:
- [ ] `.get_dummies()`
- [ ] split-apply-combine
- [ ] datetime

**Comment by deniederhut** [comment]
@davclark it might be nice to have your input on this too. The context is that we are introducing pandas in the intro series, so we might want to update the pandas workshop.

**Comment by davclark** [comment]
To be sure I'm clear - students will have basic pandas in the intro series, so here it will be intermediate / advanced topics?

**Comment by deniederhut** [comment]
Correct. I'm okay with some topical overlap or review of the basics, but it
seemed like a good opportunity to expand our offerings.

**Comment by juanshishido** [comment]
Also, this issue was started here because it was a space that was already available. When we actually create the actual intermediate / advanced materials, it will be in another repository. (I mention this in case there was confusion about why the topics mentioned in this issue were in the _introduction_ to pandas repo.)
